src/app/devices/device-toolbar/device-toolbar.component.spec.ts

```diff
@@ -6,12 +6,14 @@ import { DevicesService } from '../devices.service'
 import { DeviceToolbarComponent } from './device-toolbar.component'
 import { ActivatedRoute } from '@angular/router'
 import { of } from 'rxjs'
+import { MatDialog } from '@angular/material/dialog'
+
 describe('DeviceToolbarComponent', () => {
```

Comment:
Still a bit more to go for branches on Device Toolbar:
![image](https://user-images.githubusercontent.com/22204562/143307532-3382fd6d-479f-43c2-bdb7-0d87cad91a76.png)

Reply:
done